include/SupportedUserTypes.h

```diff
@@ -285,17 +285,31 @@ namespace ChimeraTK {
 
   template<typename NUMERIC>
   NUMERIC detail::userTypeToNumeric_impl<std::string, NUMERIC>::impl(const std::string& value) {
+    bool isHexidecimal = value.length() > 1 and (value[1] == 'x' or value[1] == 'X'); // OxABCD form hex
```

Comment:
please add parantheses around all logical blocks for clearness. I would add a check that the character before the x is a zero (actually even the comment was wrong, it had a capital "Oh".
```suggestion
    // 0xABCD form hex
    bool isHexidecimal = (value.length() > 1) and ((value[1] == 'x') or (value[1] == 'X')) and (value[0] == '0'); 
```

Reply:
I put the comment into the line above because the code because the line was getting too long for the coding style.